index.html

```diff
@@ -10,7 +10,17 @@
 </head>
 <body>
   <div class="title">
-      <h1 class="title-name">Front-End Tools</h1>
-  </div>  
+    <h1 class="title-name">Front-End Tools</h1>
+  </div>
+  <div class="tech">
```

Comment:
Is this div neccesary? can't we group it with its only child?

Reply:
Yes, its not necessary in this task. I done it thinking in next implementations.